services/src/main/java/org/fao/geonet/api/OpenApiController.java

```diff
@@ -76,22 +84,21 @@ public class OpenApiController extends AbstractOpenApiResource {
 
     private final Optional<SecurityOAuth2Provider> springSecurityOAuth2Provider;
 
-    private final Optional<RouterFunctionProvider> routerFunctionProvider;
+    private final List<RouterOperation> routerOperations;
```

Comment:
Apparently is not use anywhere?

Reply:
This was previously there as unused as well.  I was not sure if someone had added there with the intent to use it at a later time?

I will remove it.